crates/toml_edit/src/internal_table.rs

```diff
@@ -0,0 +1,179 @@
+use crate::{InlineTable, Item, Key, Table, Value};
+/// a place for helper methods supporting the table-like impls
+
+use crate::table::KeyValuePairs;
+
+/// GetTableValues provides the logic for displaying a table's items in their parsed order
+pub(crate) trait GetTableValues {
```

Comment:
It looks like this is a roundabout way to share code for manipulating `KeyValuePairs`.  Seems like the functionality should just be writtenm for `KeyValuePairs` and shared

imo if I were to do this, I would 
- Move `KeyValuePairs` into here
- Either rename the file to be about `KeyValuePairs` or rename the type (though not a fan of `InternalTable` though `InnerTable` might work)
- Either make `get_values` and `sort_values` either non-member functions that take `KeyValuePairs` or make `KeyValuePairs` a newtype (which might add its own complexity)

Reply:
Feel free to split this refactor out into its own commit.  It'll likely make it easier to review and get merged which can speed up reviewing both PRs.

Also, please consider if there are intermediate steps to the above that should be broken out into their own commit (e.g. adding a newtype of we go that route)